src/Backend/Model_Server/.gitignore

```diff
@@ -160,3 +160,5 @@ cython_debug/
 #.idea/
 
 local-test.cmd
+
+letter_model.pt
```

Comment:
Moving the `.gitignore` from `Backend` to `Backend/Model_Server` doesn't look correct to me.